common/share widget/reports/pre-ally-1.json

```
@@ -0,0 +1,589 @@
+{
+    "@context": "https://wet-boew.github.io/vocab/context/2023/assessment-report-en.json-ld",
+
+    "@type": ["earl:Assertion", "acr:AssessmentReport"],
+
+    "earl:subject": {
+        "@id": "_:subject",
+        "dct:references": "https://wet-boew.github.io/wet-boew/demos/share/share-en.html",
+        "@type": [
+            "earl:TestSubject",
+            "schema:WebPage"
+        ],
+        "dct:description": "Accessibility conformance reports of Share widget",
+        "earl:pointer": {
+            "@type": "oa:CssSelector",
+            "@value": "share"
```

The CSS selector for this plugin is: `.wb-share`

```suggestion
            "@value": ".wb-share"
```